### ISSUE TITLE
[Configurable] JS error on product page

### ISSUE THREAD

**Issue by ihor-sviziev** [magento/magento2#25601]
### Preconditions (*)
1. Magento 2.3.3 or 2.3-develop

### Steps to reproduce (*)
1. Create Configurable product
2. Go to this product on frontend
3. Add this product to cart,
![image](https://user-images.githubusercontent.com/1873745/68857800-494c3100-06ec-11ea-84e1-e9bc1cfa05c6.png)
4. In the mini cart or on the cart page click on "Edit" (![image](https://user-images.githubusercontent.com/1873745/68857717-0ee29400-06ec-11ea-8ba7-1797acfa9e4e.png)) icon
5. Open Google Chrome Developer tools

### Expected result (*)
1. You should not see any JS errors
2. Already selected configurable option should be selected 

### Actual result (*)
1. We have JS error 
```
Uncaught TypeError: Cannot read property 'find' of undefined
    at $.<computed>.<computed>.updateMsrpPrice (msrp.js:355)
    at $.<computed>.<computed>.updateMsrpPrice (widget.js:100)
    at $.<computed>.<computed>.onUpdateMsrpPrice (msrp.js:333)
    at $.<computed>.<computed>.onUpdateMsrpPrice (widget.js:100)
    at HTMLDocument.dispatch (jquery.js:5232)
    at HTMLDocument.elemData.handle (jquery.js:4884)
    at Object.trigger (jquery.js:5136)
    at Object.jQuery.event.trigger (jquery-migrate.js:633)
    at HTMLDocument.<anonymous> (jquery.js:5866)
    at Function.each (jquery.js:376)
```
![image](https://user-images.githubusercontent.com/1873745/68857848-697bf000-06ec-11ea-802d-b4883bff52f0.png)
2. Already selected configurable option is not selected

**Comment by ihor-sviziev** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, here is your [Magento instance](https://i-25601-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25601-2-3-develop.instances.magento-community.engineering/admin](https://i-25601-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
@krishprakash Thank you for verifying the issue.

Unfortunately, not enough information was provided to acknowledge ticket. Please consider adding the following:
- [ ] Add `"Component: "` label(s) to this ticket based on verification result. If uncertain, you may follow the best guess

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @krishprakash 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-23138` were created

**Issue Available**: @krishprakash, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev. Thank you for your report. 
The issue has been fixed in magento/magento2#25606 by @ihor-sviziev in 2.3-develop branch
Related commit(s):
 - [d26f8dad182b6f2cb8608758989fff3edb53a09f](https://github.com/magento/magento2/commit/d26f8dad182b6f2cb8608758989fff3edb53a09f)

The fix will be available with the upcoming 2.3.5 release.
